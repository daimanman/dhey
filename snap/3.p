{"C":10,"Z":"2","N":10,"Remark":"TEST TASK","Url":"","Method":"post","TaskId":3,"Type":"","FileId":"1","Status":0,"Err":"","QH":{}}