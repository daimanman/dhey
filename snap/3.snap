{"AvgTotal":0.8121862999999999,"Fastest":0.0561479,"Slowest":0.1614107,"Average":0.08121862999999999,"Rps":12.31244604840047,"AvgConn":0.00582258,"AvgDNS":0.0008005699999999999,"AvgReq":0,"AvgRes":0.022540159999999997,"AvgDelay":0.052855889999999996,"ConnMax":0,"ConnMin":0.0080057,"DnsMax":0,"DnsMin":0.0080057,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0.0039724,"DelayMax":0.0512478,"DelayMin":0.0571818,"Lats":[0.1614107,0.100273,0.0571471,0.1002715,0.0561479,0.0571533,0.0992636,0.0611593,0.0611621,0.0581978],"ConnLats":[0.0080057,0,0,0,0,0,0,0,0,0],"DnsLats":[0.0080057,0,0,0,0,0,0,0,0,0],"ReqLats":[0,0,0,0,0,0,0,0,0,0],"ResLats":[0.050133,0.0490252,0.0040099,0.0481324,0.0039724,0.0050131,0.0471015,0.0039775,0.0089822,0.0050544],"DelayLats":[0.0530519,0.0512478,0.0531372,0.0521391,0.0521755,0.0521402,0.0521621,0.0571818,0.0521799,0.0531434],"Total":812186300,"ErrorDist":{},"StatusCodeDist":{"200":10},"SizeTotal":0,"SizeReq":0,"NumRes":10,"LatencyDistribution":[{"Percentage":10,"Latency":0.0571471},{"Percentage":25,"Latency":0.0581978},{"Percentage":50,"Latency":0.0611621},{"Percentage":75,"Latency":0.100273},{"Percentage":90,"Latency":0.1614107},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.0561479,"Count":1,"Frequentcy":0.1},{"Mark":0.06667418,"Count":5,"Frequentcy":0.5},{"Mark":0.07720046,"Count":0,"Frequentcy":0},{"Mark":0.08772674,"Count":0,"Frequentcy":0},{"Mark":0.09825302,"Count":0,"Frequentcy":0},{"Mark":0.1087793,"Count":3,"Frequentcy":0.3},{"Mark":0.11930558,"Count":0,"Frequentcy":0},{"Mark":0.12983186,"Count":0,"Frequentcy":0},{"Mark":0.14035814,"Count":0,"Frequentcy":0},{"Mark":0.15088442,"Count":0,"Frequentcy":0},{"Mark":0.1614107,"Count":1,"Frequentcy":0.1}],"TaskId":3,"Duration":0.8121863}